web/src/hooks/queries/useValidation.ts

```diff
@@ -25,7 +32,8 @@ const runValidation = async (pep: FileList | string | undefined, schema: string
       formData.append('pep_files', pep_files[i]);
     }
   }
-  formData.append('schema', schema || '');
+  formData.append('schema', params.schema || '');
```

Comment:
does `FormData` not allow appending `undefined`? I'd think we'd want to send `undefined` (or at least `null`) to the server so it gets `None`. We check for `None` server-side, so I was thinking we'd send `None` here instead of an empty string (`''`) if the `params.schema` or `params.schema_registry` is `undefined`.

Reply:
Ah I see, yea it seems to want to fight me if I try to use None or undefined instead of an empty string